docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -142,13 +143,25 @@ $ echo $?
 # 1
 ```
 
-When logging in, the `nginxrestarter` user and any groups that do not already exist
-will be created on the host. The `nginxrestarter` user will be added to the `ubuntu`,
-`nginx` and `other` groups as specified in the `host_groups` field.
-
-A file will also be created in `/etc/sudoers.d` with the contents of the
-`host_sudoers` file written with one entry per line, each prefixed with the username
-of the user that has logged in.
+When the user above logs in, the `nginxrestarter` user and any groups that do
+not already exist will be created on the host. The `nginxrestarter` user will be
+added to the `ubuntu`, `nginx` and `other` groups as specified in the
+`host_groups` field.
+
+The Teleport SSH Service executes `useradd` in order to create new users on the
+host, and returns an error if it cannot find the `useradd` binary. It creates a
+new home directory based on the host user's name and adds the user to the groups
```

Comment:
```suggestion
new home directory for the user with  the user's name and adds the user to the groups
```

just a minor suggestion.

Reply:
Aside from the double-space in the suggestion, "based on" may or may not be more appropriate. Does `useradd` always create a home directory with the exact name as the user?

Reply:
It uses the `--create-home` flag of `useradd` (https://github.com/gravitational/teleport/blob/1441d81d152be7cb64e91614fd1f7a5f1ee4e1a2/lib/utils/host/hostusers.go#L58), set to the name of the host user